samples/arch/mpu/mpu_test/README.rst

```diff
@@ -1,11 +1,11 @@
 .. _mpu_test:
 
-Memory Protection Unit (MPU) TEST
+Memory Protection Unit (MPU) SAMPLE
```

Comment:
Sample

Reply:
Did not get? Do you mean changing the name of mpu_test to mpu_sample?

Reply:
No, there is no need for uppercase here.